semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/ConfigOps.scala

```diff
@@ -160,12 +160,14 @@ sealed abstract class SymbolMode {
   def name: String = toString.toLowerCase
   def saveDefinitions: Boolean = this == All || this == Definitions
   def saveReferences: Boolean = this == All
+  def saveMaterializedTypes: Boolean = this == AllWithMaterializedTypes
 }
 object SymbolMode {
   def name: String = toString.toLowerCase
   def unapply(arg: String): Option[SymbolMode] =
     all.find(_.toString.equalsIgnoreCase(arg))
   def all = List(All, Definitions, None)
+  case object AllWithMaterializedTypes extends SymbolMode
```

Comment:
Let's introduce a dedicated `OccurrenceMode`. `SymbolMode` traditionally only affects the `SymbolInformation` section in SemanticDB.